src/components/PrimaryColorBtn.jsx

```diff
@@ -0,0 +1,54 @@
+import styled, { css } from 'styled-components'
+
+import { getBreakpointsStyles } from '../styles/breakpoints'
+import { lightTheme, darkTheme } from '../styles/theme/themes'
+
+export const PrimaryColorBtn = ({ children, isDefaultShadow = false, theme }) => {
+  return (
+    <PrimaryBtn isDefaultShadow={isDefaultShadow} theme={theme}>
+      {children}
+    </PrimaryBtn>
+  )
+}
+
+const PrimaryBtn = styled.button`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-color: ${(prop) =>
+    prop.theme === 'light' ? lightTheme.colors.primary : darkTheme.colors.primary};
```

Comment:
Unfortunatelly you are using values from theme in the wrong way.

`theme.colors.primary` would be enought, no need to use condition for it. Theme will have correct values depends on selected theme type automatically.

Please fix it everywhere